lib/drawer/index.js

```diff
@@ -0,0 +1,50 @@
+import d3 from 'd3';
+import { delimiters } from './delimiters';
+import filterData from '../filterData';
+import { init as initMetaballs } from '../metaballs';
+
+export default function (svg, dimensions, scales, configuration) {
```

Comment:
Please exand configuration, as signature would be really meaningful, please, please